pkg/apparmor/apparmor_linux.go

```diff
@@ -25,9 +26,34 @@ var profileDirectory = "/etc/apparmor.d"
 // IsEnabled returns true if AppArmor is enabled on the host.
```

Comment:
The comment needs updating to account for the additional checks

Reply:
Fixed the comment :+1: